go/libkb/home.go

```diff
@@ -120,7 +122,11 @@ func (d Darwin) homeDir(dirs ...string) string {
 	return filepath.Join(dirs...)
 }
 
-func (d Darwin) CacheDir() string                 { return d.homeDir(d.Home(false), "Library", "Caches") }
+func (d Darwin) CacheDir() string { return d.homeDir(d.Home(false), "Library", "Caches") }
+func (d Darwin) SandboxCacheDir() string {
+	// The container name "keybase" is the group name specified in the entitlement for sandboxed extensions
+	return d.homeDir(d.Home(false), "Library", "Group Containers", "keybase", "Library", "Caches")
```

Comment:
Yeah lowercase keybase is the entitlement group name... this is normal, conventional that the entitlement group is lowercase